components/Cards/card.css

```diff
@@ -0,0 +1,156 @@
+/* card */
+
+.card {
+  margin: 1rem 2rem;
+  max-width: 300px;
+  position: relative;
+  border-radius: 1rem;
+  background-color: #fff;
+  padding: 1rem;
+  border-radius: 1rem;
+  box-shadow: var(--secondary-box-shadow);
+}
+
+.bdge-sm {
+  position: absolute;
+  font-size: 1.2rem;
+  font-weight: 400;
+  padding: 0.2rem 0;
+  top: 2rem;
+  width: 100px;
+  padding: 0.3rem;
+  clip-path: polygon(0% 0%, 71% 0%, 53% 50%, 69% 97%, 0 97%);
+}
+
+.bgde-lg:after {
+  content: "Most Popular";
+  font-size: 0.8rem;
+  text-align: center;
+  line-height: 20px;
+  transform: rotate(45deg);
+  width: 120px;
+  background-color: var(--dk-pink);
+  color: var(--lt-pink);
+  display: block;
+  position: absolute;
+  top: 1.6rem;
+  right: -1.5rem;
+  clip-path: polygon(
+    120px 20px,
+    90px -10px,
+    30px -10px,
+    0px 20px,
+    10px 30px,
+    110px 30px
+  );
+}
+
+.bdge-cl {
```

Comment:
naming consistency can be considered here as well. 3 different names are bdg- , badge-, bdge-